### ISSUE TITLE
Aggressively renaming strings of rule `comment-content` when not needed.

### ISSUE THREAD

**Issue by lexa-uw** [sindresorhus/eslint-plugin-unicorn#3613]
Rule `comment-content` affect on JSDoc `@params` attributes.

eslint.config.mjs
```js
  import unicorn from 'eslint-plugin-unicorn'

  export default [
    {
      files: ['**/*.js'],
      plugins: { unicorn },
      rules: {
        'unicorn/comment-content': 'error',
      },
    },
  ]
```
input.js
```js
  /**
   * Renders the requested page.
   * @param url - Requested path handled by the router.
   * @returns Rendered markup.
   */
  export const render = (url) => url.toUpperCase()
```

`url` argument should be lowercase in function and in JSDoc block in `@param/@return/etc`. 

But rule renames/alerts everytime.

Of course it should be uppercase in other cases as expected.

Could you make ignorable JSDoc block by default?
